tex/alg-geom/affine-var.tex

```diff
@@ -84,7 +84,7 @@ \section{Naming affine varieties via ideals}
 	Convince yourself that $\VV(I) = \{(3,4)\}$.
 \end{ques}
 So rather than writing $\VV(x-3, y-4)$ it makes sense to
-think about this as $\VV\left( I \right)$, where $I = (x-3,y-4)$ is the \emph{ideal}
+think about this as $\VV( I )$, where $I = (x-3,y-4)$ is the \emph{ideal}
```

Comment:
Better spacing this way. https://tex.stackexchange.com/q/173717/250119 , but maybe it's clearer to look at the PDF output.